physlr/physlr.py

```diff
@@ -1519,7 +1566,7 @@ def merge_communities(g, communities, node_set=0, strategy=0, cutoff=20):
         if len(communities) == 1 and (node_set == 0 or strategy != 1):
             return communities
         if strategy == 1:  # Merge by Initializing Louvain with the communities
-            return Physlr.detect_communities_louvain(g, node_set, communities)
+            return Physlr.detect_communities_louvain(g, communities, node_set)
```

Comment:
`    def detect_communities_louvain(g, node_set, init_communities=None):` Your function definition looks like this